manual/src/tutorials/parallelism.etex

```diff
@@ -737,5 +738,5 @@ If the "compare_and_set" fails, then some other domain is also attempting to
 update the atomic reference at the same time. In this case, the "push" and
 "pop" operations call "Domain.cpu_relax" to back off for a short duration
 allowing competing domains to make progress before retrying the failed
-operation. This lock-free stack implementation is also known as Treiber
-stack.
+operation. This lock-free stack implementation is also known as the 
```

Comment:
I'd personally put "a" rather than "the", but I'm actually wondering if the whole sentence is not quite there - is the point more that the lock-free stack _is_ a Treiber stack, rather than being known as one?

Reply:
How about "this lock-free stack implementation is called a Treiber Stack"?